src/devices/Ads1115/Ads1115.cs

```diff
@@ -18,8 +20,22 @@ public class Ads1115 : IDisposable
 
         private InputMultiplexer _inputMultiplexer;
 
+        private MeasuringRange _measuringRange;
+
+        private DataRate _dataRate;
+        private DeviceMode _deviceMode;
+
+        private GpioController _gpioController;
+        private int _alrtRdyPin;
```

Comment:
nit: please do not use abbreviations in variable names (with some super common exceptions like `i`, `id`, `xml`, etc)

Reply:
This is how the pin is called in the documentation. And also the text on my breakout board reads "ALRT"

Reply:
Changed, since its the pin number on the gpio side anyway, not the number of the pin on the ADS